src/cluster.h

```diff
@@ -23,6 +23,7 @@
 #define CLUSTER_SLOTS (1<<CLUSTER_SLOT_MASK_BITS) /* Total number of slots in cluster mode, which is 16384. */
 #define CLUSTER_SLOT_MASK ((unsigned long long)(CLUSTER_SLOTS - 1)) /* Bit mask for slot id stored in LSB. */
 #define INVALID_CLUSTER_SLOT (-1) /* Invalid slot number. */
+#define GETSLOT_CROSSSLOT  (-2)
```

Comment:
can we rename these to something like `CLUSTER_INVALID_SLOT` and `CLUSTER_CROSSSLOT` to make their names a bit more consistent and reflect that they are related?